tokio/src/io/read_buf.rs

```diff
@@ -270,6 +273,25 @@ impl<'a> ReadBuf<'a> {
     }
 }
 
+#[cfg(feature = "bytes")]
+unsafe impl<'a> BufMut for ReadBuf<'a> {
```

Comment:
Instead of the cfg import at the top, just do like this.
```suggestion
unsafe impl<'a> bytes::BufMut for ReadBuf<'a> {
```